tests/apps/crunchy/test_crunchy.py

```diff
@@ -107,14 +111,38 @@ def test_is_fastq_compression_possible_spring_exists(
     assert spring_file.exists()
 
     # WHEN checking if SPRING compression is done
-    result = crunchy_api.is_fastq_compression_possible(compression_object)
+    result = crunchy_api.is_fastq_compression_possible(
+        compression_obj=compression_object, is_spring_archived=False
+    )
 
     # THEN result should be False since the compression flag exists
     assert result is False
     # THEN assert that the correct message was communicated
     assert "SPRING file found" in caplog.text
 
 
+def test_is_fastq_compression_possible_spring_is_archived(
+    crunchy_config: dict[str, dict[str, Any]], compression_object: CompressionData, caplog
+):
+    """Test if FASTQ compression is possible when the Spring file is archived
+
+    This means that the SPRING file exists
+    """
+    caplog.set_level(logging.DEBUG)
+    # GIVEN a crunchy-api, and existing FASTQ paths
```

Comment:
it would be nice to make an assertion that the spring file is archived here, if possible. If it is too much work, disregard

Reply:
Well this test runs a method which takes whether the spring file as input. So there is no actual assertion that can be made without some rework.